test/e2e_node/runner/remote/run_remote.go

```diff
@@ -466,18 +461,14 @@ type imageObj struct {
 	name         string
 }
 
-func (io imageObj) string() string {
-	return fmt.Sprintf("%q created %q", io.name, io.creationTime.String())
-}
-
 type byCreationTime []imageObj
 
 func (a byCreationTime) Len() int           { return len(a) }
 func (a byCreationTime) Less(i, j int) bool { return a[i].creationTime.After(a[j].creationTime) }
 func (a byCreationTime) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 
-// Returns a list of image names based on regex and number of previous images requested.
-func getGCEImages(imageRegex, imageFamily string, project string, previousImages int) ([]string, error) {
+// Returns a list of image names based on regex and given GCE project.
+func getGCEImages(imageRegex, imageFamily string, project string) ([]string, error) {
```

Comment:
Given the changes being made the function signature should also change 
```suggestion
func getGCEImages(imageRegex, imageFamily string, project string) (string, error) {
```